Detections/Ubiquiti/UbiquitiDestinationInTiList.yaml

```diff
@@ -0,0 +1,39 @@
+id: db60ca0b-b668-439b-b889-b63b57ef20fb
+name: Ubiquiti - Connection to known malicious IP or C2
+description: |
+  'Detects allowed connections to IP addresses which are in TI list and are known to be malicious.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: UbiquitiUnifi
+    dataTypes:
+      - Ubiquiti_CL
+queryFrequency: 1h
+queryPeriod: 14d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Exfiltration
+  - CommandAndControl
+relevantTechniques:
+  - T1071
+  - T1571
+  - T1572
+query: |
+  let ioc_lookback = 14d;

```

Comment:
Not needed for the detection as you already have 14d specified for the queryPeriod.  This will be used instead of lookback.

Reply:
updated